.travis.yml

```diff
@@ -1,4 +1,5 @@
 services: [ docker ]
+language: generic
```

Comment:
why do we need this ?

Reply:
If we don't give this option it assumes that it is ruby and gives error

Reply:
Apparently Travis behaviour changed. We may need similar fixes on other repos as well. The default language is ruby on Travis and it complains because of a missing Rakefile (that stores dependency information on ruby projects)